packages/snaps-cli/src/cli.ts

```diff
@@ -7,6 +7,7 @@ export function cli(argv: string[], commands: any): void {
   const rawArgv = argv.slice(2);
   // eslint-disable-next-line @typescript-eslint/no-unused-expressions
   yargs(rawArgv)
+    .scriptName('mm-snap')
```

Comment:
This ensures that the name of the script from `yargs`'s perspective is always `mm-snap`. If we don't do this, the script name changes depending on how you execute it, e.g. `node dist/main.js` will make `--help` show `main.js` as the script name.